net/netfilter/nf_tables_api.c

```diff
@@ -9032,11 +9064,21 @@ static void __net_exit nf_tables_pre_exit_net(struct net *net)
 
 static void __net_exit nf_tables_exit_net(struct net *net)
 {
+	struct nftables_pernet *nft_net = nft_pernet(net);
```

Comment:
This change comes in as a part of a larger change that Upstream Did not pull in:
`netfilter: nf_tables: use net_generic infra for transaction data`
https://github.com/torvalds/linux/commit/0854db2aaef3fcdd3498a9d299c60adea2aa3dc6#diff-233a9dea2c513f5d208bba55e0f8ef4e8a29f5c584d30b5815cfa6c2cf5361ecR8502-R8505

With this we also need to change the `mutex_unlock` as well
```
-	mutex_unlock(&net->nft.commit_mutex);
+	mutex_unlock(&nft_net->commit_mutex);
```

Reply:
This is one is along the same lines as your previous comment above.  I think the code I have matches 534 and given the context of the code around it I don't think it's the right thing to do.

I have an updated patch series ready to push, needs some more testing.  Especially to make sure the nftables tests still pass.

Reply:
Yeah this was me not seeing the pointer versus the `.` accessor, this is the 3-way comparison review problem.